soc/xtensa/intel_adsp/cavs_v25/Kconfig.defconfig.series

```diff
@@ -15,8 +15,9 @@ config SOC
 	string
 	default "intel_cavs_25"
 
+# Hardware has four cores, limited to two pending test fixes
```

Comment:
A link to https://github.com/zephyrproject-rtos/zephyr/pull/40960 woud be useful here.